### PR TITLE
*: add OWNERS file

### PR DESCRIPTION
**Pull request by abhinavdahiya**

### CODE REVIEW

File: cmd/machine-config-daemon/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+  - ashcrow
```
Comment (ashcrow) [inline cmd/machine-config-daemon/OWNERS]: Can we get @jlebon added here as well? He also will be highly involved in this.

File: pkg/daemon/OWNERS [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+  - ashcrow
```
Comment (ashcrow) [inline pkg/daemon/OWNERS]: Can we get @jlebon here as well?

File: cmd/machine-config-daemon/OWNERS (hunk 1 above)
Comment (abhinavdahiya) [reply cmd/machine-config-daemon/OWNERS]: :+1: done

File: pkg/daemon/OWNERS (hunk 2 above)
Comment (abhinavdahiya) [reply pkg/daemon/OWNERS]: :+1:  done

**Pull request merged**